assets/blocks/single-course.scss

```diff
@@ -1,4 +1,4 @@
 
-@import 'button/button.scss';
-@import 'course-outline/style.scss';
-@import 'course-progress/style.scss';
+@import 'take-course/take-course';
```

Comment:
I'd prefer to start naming these files a bit more unique, because it's getting hard to navigate the code when everything is called `style.scss` and `edit.js`. This way it's possible to just start typing take-course when opening a file to find it. I'll write up a proposal so we can settle on a convention.